README.md

```diff
@@ -12,48 +12,76 @@ Backends
 
 - OpenCV (in-development)
 
+
+Features
+
+- Easy and hassle free (read/write with ```for-loop```, ```with-block```, ```while-loop```)
+- Batch Support. Read and write frames in batches.
+
+
 # Read Video
 
+
+## Read with ```for-loop```
 ```python
 from vidsz.opencv import Reader
 
 # open reader
 reader = Reader("static/countdown.mp4")
 
-# print info: width, height, fps etc.
-print(reader)
+# read frame with for loop
+for frame in reader:
+    # use frame however you like
```

Comment:
```suggestion
    # use ndarry-frame however you like
```

Reply:
Picked it up from the comment on line 54. I think it can be useful information, what do you think?